app/validators/email_validator.rb

```diff
@@ -0,0 +1,7 @@
+class EmailValidator < ActiveModel::EachValidator
```

Comment:
Remove this file - already exists in `publisher.rb`

Reply:
Where do you see this?